src/lib/charts/chart.helpers.tsx

```diff
@@ -0,0 +1,20 @@
+import { ChartTooltipModel } from 'chart.js';
+import React from 'react';
+import { RefObject } from 'react';
+import { Bar, Pie } from 'react-chartjs-2';
+import ReactDOM from 'react-dom';
+import { ChartTooltip } from './ChartTooltip/ChartTooltip';
+
+export type ChartLabels = (string | number | string[] | number[] | Date | Date[] | moment.Moment | moment.Moment[])[];
+
+// Note: Abstracted from https://github.com/reactchartjs/react-chartjs-2/issues/151#issuecomment-470282163
+export const customTooltip = (tooltipModel: ChartTooltipModel, chartRef: RefObject<Pie | Bar>) => {
+  let tooltipEl = document.getElementById('lc-chart-tooltip-wrapper');
+  if (!tooltipEl) {
+    tooltipEl = document.createElement('div');
+    tooltipEl.id = 'lc-chart-tooltip-wrapper';
+    document.body.appendChild(tooltipEl);
+  }
+
+  ReactDOM.render(<ChartTooltip model={tooltipModel} chartRef={chartRef} />, tooltipEl);
```

Comment:
@jaredhill4 this seems to work a lot better than trying to use a portal and trying to re-render everything that way. kind of a best of both worlds, because we are still using a custom react component and now we don't have to have it in each file.

Reply:
Haha. I actually had a version that used `ReactDOM.render` first, because I couldn't get the portal to work, but then once I figured out the portal, I moved away from `ReactDOM.render`.

Reply:
Yea, this looks very similar to what I had originally. I was just trying to avoid having to pull out into a helper function.

We didn't have to have it in each file. We could have created a `ChartBase` or something like that, that each used, but I suppose this works, too. It 6 one way, 1/2 dozen the other to me, I suppose.

Reply:
This way worked a lot better for me. When I was getting deep into things, there was a lag and it wasn't as good. When I switched over to this solution, it was a lot better.